src/EnergyPlus/HVACManager.cc

```diff
@@ -2995,4 +2904,46 @@ void CheckAirLoopFlowBalance(EnergyPlusData &state)
     }
 }
 
+void ConvergenceErrors(EnergyPlusData &state,
+                       std::array<bool, 3> HVACNotConverged,
+                       std::array<Real64, 10> DemandToSupply,
+                       std::array<Real64, 10> SupplyDeck1ToDemand,
+                       std::array<Real64, 10> SupplyDeck2ToDemand,
```

Comment:
Hmmmm...  are these going to be copied?  This is something that might actually have to be changed.  @lgu1234 I will likely make a pass at cleaning this up and maybe touch some of the other random notes I've made here.